hooks/useHardwareWallet.js

```diff
@@ -104,7 +104,7 @@ export function useLedger({ onAccountChosen }) {
         }
       });
     },
-    [show]
+    [accountSelection, show]
```

Comment:
@tyler17 I traced the null `maker` error to `onAccountChosen` in `WalletManager.js`. Adding this dependency gives us access to `maker` in that function & now ledger seems to be working. I wasn't sure if there was more you wanted to do with this branch, so I didn't merge it in yet.

Reply:
@b-pmcg Great, thanks!  If everything looks good, we can merge, I was planning on just testing everything a little more.